src/controller/CHIPCluster.h

```diff
@@ -263,20 +264,23 @@ class DLL_EXPORT ClusterBase
     CHIP_ERROR
     SubscribeAttribute(void * context, ReadResponseSuccessCallback<typename AttributeInfo::DecodableArgType> reportCb,
                        ReadResponseFailureCallback failureCb, uint16_t minIntervalFloorSeconds, uint16_t maxIntervalCeilingSeconds,
-                       SubscriptionEstablishedCallback subscriptionEstablishedCb = nullptr, bool aIsFabricFiltered = true,
+                       SubscriptionEstablishedCallback subscriptionEstablishedCb = nullptr,
+                       ResubscriptionAttemptCallback ResubscriptionAttemptCb = nullptr, bool aIsFabricFiltered = true,
                        bool aKeepPreviousSubscriptions = false, const Optional<DataVersion> & aDataVersion = NullOptional)
     {
         return SubscribeAttribute<typename AttributeInfo::DecodableType, typename AttributeInfo::DecodableArgType>(
             context, AttributeInfo::GetClusterId(), AttributeInfo::GetAttributeId(), reportCb, failureCb, minIntervalFloorSeconds,
-            maxIntervalCeilingSeconds, subscriptionEstablishedCb, aIsFabricFiltered, aKeepPreviousSubscriptions, aDataVersion);
+            maxIntervalCeilingSeconds, subscriptionEstablishedCb, ResubscriptionAttemptCb, aIsFabricFiltered,
+            aKeepPreviousSubscriptions, aDataVersion);
     }
 
     template <typename DecodableType, typename DecodableArgType>
     CHIP_ERROR SubscribeAttribute(void * context, ClusterId clusterId, AttributeId attributeId,
                                   ReadResponseSuccessCallback<DecodableArgType> reportCb, ReadResponseFailureCallback failureCb,
                                   uint16_t minIntervalFloorSeconds, uint16_t maxIntervalCeilingSeconds,
                                   SubscriptionEstablishedCallback subscriptionEstablishedCb = nullptr,
-                                  bool aIsFabricFiltered = true, bool aKeepPreviousSubscriptions = false,
+                                  ResubscriptionAttemptCallback ResubscriptionAttemptCb = nullptr, bool aIsFabricFiltered = true,
```

Comment:
Why is this new argument name pascal case, while the rest are camel case?

Reply:
typo error, fixed, thanks for catch up.